test/integration/patients/worklist/schedule.js

```diff
@@ -1709,6 +1929,12 @@ context('schedule page', function() {
       .get('@listSearch')
       .type('Second Action');
 
+    cy
+      .get('[data-count-region] .schedule__count-total')
+      .should(el => {
+        expect(el.text().trim()).to.equal('1 Action');
```

Comment:
why is the trim necessary?  `11 Actions`?
could maybe be `should('contain', '1 Action').should('not.contain', 'Actions')`

Reply:
I'm not sure why the `trim()` is/was necessary, it had a bunch of spaces added when Cypress was comparing them.

But `should('contain', '1 Action').should('not.contain', 'Actions')` is a better way to do it. So I switched it over to that instead.